study-builder/studies/rarex/consent-lar-assent.conf

```diff
@@ -486,6 +486,66 @@
           "blockType": "QUESTION",
           "shownExpr": null
         },
+        {
+          "question": {
+            include required("../snippets/text-question-signature-required.conf"),
+            "stableId": "LAR_CONSENT_ASSENT_PATIENT_SIGNATURE",
```

Comment:
Why do we need to add this again? We already have this in section 4 below.

Reply:
Definitely, this is a mistake, I should have written something like "LAR_CONSENT_ASSENT_CHILD_SIGNATURE" since the signature belongs to the child

Reply:
Actually for the assents we have a single representative pdf file, it's called  "assent-child-7-17-adults-unable-consent.pdf", you may find it in studies/rarex/pdfs/..

as it is shown there, we need to include patient signature 2 times. I'll try to explain and pls correct me if I'm wrong

first time -> signature of patient who is unable to consent

second time -> signature of patient who is unable to consent and is able to read assent

Reply:
@yufengwng Did you mean to change stableId-s?

Reply:
Okay, I see what you're saying. So it looks like we'll need 2 sets of these questions. I previously removed one set of them, sorry about that. So there's 2 different scenarios here:

First scenario: child is able to read.
* User answers "Yes" in the UI assent question.
* Then, the UI should show the "able read" set of questions and hide the "not able" set of questions.

Second scenario: child is not able to read.
* User answers "No" in the UI assent question.
* * Then, similarly the UI shows one set of questions and hide the other set.

The PDF should be configured with BOTH sets of questions, and this is okay because only one set of questions should have been answered in the UI. So when we go generate the PDF, only one section in the `assent...unable-consent.pdf` file will have the signatures.

So yes, you're right, we need to add back in another set of these questions with different stable ids and show/hide logic.